src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -88,11 +90,26 @@ void initializeReads() {
         if (hasReference()){
             // pass in reference if available, because CRAM files need it
             factory = SamReaderFactory.makeDefault().validationStringency(ValidationStringency.SILENT).referenceSequence(referenceArguments.getReferenceFile());
+        } else if (hasCramInput()) {
+            throw new UserException("A reference file is required when using CRAM files.");
         }
         reads = ! readArguments.getReadFiles().isEmpty() ? new ReadsDataSource(readArguments.getReadFiles(), factory) : null;
     }
 
     /**
+     * Helper method that simply returns a boolean regarding whether the input has CRAM files or not.
+     */
+    private boolean hasCramInput() {
+        boolean hasCram = false;
+        for (File inputFile : readArguments.getReadFiles()) {
+            if (FilenameUtils.getExtension(inputFile.getName()).equalsIgnoreCase("cram")) {
+                hasCram = true;
+            }
+        }
+        return hasCram;
```

Comment:
This method would be a bit better without the boolean variable. Eg.,

        for (File inputFile : readArguments.getReadFiles()) {
            if (FilenameUtils.getExtension(inputFile.getName()).equalsIgnoreCase("cram")) {
                return true;
            }
        }
        return false;